server/views/pages/paginatedPersonSearchResults.njk

```diff
@@ -24,56 +23,73 @@
+            </p>
+            <p class="govuk-body-l govuk-!-margin-bottom-3">
+                <span class='govuk-!-padding-right-2 govuk-!-font-weight-bold'>First name:</span>
+                <span>{{ firstName }}</span>
+            </p>
+            {% if user.hasPpcsRole %}
+                <a href="/find-a-recall-request-by-name" class="govuk-body-l govuk-link">Change <span class="govuk-visually-hidden">search term</span></a>
+            {% else %}
+                <a href="/search-by-name" class="govuk-body-l govuk-link">Change <span class="govuk-visually-hidden">search term</span></a>
+            {% endif %}
         {% endif %}
     </p>
 
     {% if page.results.length %}
-    <table class="govuk-table">
-        <caption class="govuk-table__caption govuk-visually-hidden">Persons found</caption>
-        <thead class="govuk-table__head">
-        <tr class="govuk-table__row">
-            <th scope="col" class="govuk-table__header">Name</th>
-            <th scope="col" class="govuk-table__header">CRN</th>
-            <th scope="col" class="govuk-table__header">Date of birth</th>
-        </tr>
-        </thead>
-        <tbody class="govuk-table__body">
-        {% for person in page.results %}
-            <tr class="govuk-table__row" data-qa='row-{{ person.crn }}'>
-                <td class="govuk-table__cell">
-                    {% if person.userExcluded %}
+        <table class="govuk-table">
+            <caption class="govuk-table__caption govuk-visually-hidden">Persons found</caption>
+            <thead class="govuk-table__head">
+                <tr class="govuk-table__row">
+                    <th scope="col" class="govuk-table__header">Name</th>
+                    <th scope="col" class="govuk-table__header">CRN</th>
+                    <th scope="col" class="govuk-table__header">Date of birth</th>
+                </tr>
+            </thead>
+            <tbody class="govuk-table__body">
+                {% for person in page.results %}
+                    <tr class="govuk-table__row" data-qa='row-{{ person.crn }}'>
+                        <td class="govuk-table__cell">
+                            {% if person.userExcluded %}
                         You are excluded from viewing this case
                     {% elseif person.userRestricted %}
                         This is a restricted case
                     {% endif %}
-                    <a href='/cases/{{ person.crn }}/overview' data-qa='name'>{% if person.userExcluded or person.userRestricted %}(read more information){% else %}{{ person.name }}{% endif %}</a>
-                </td>
-                <td class="govuk-table__cell">{{ person.crn }}</td>
-                <td class="govuk-table__cell">{{ formatDateTimeFromIsoString({isoDate: person.dateOfBirth}) }}</td>
-            </tr>
-        {% endfor %}
-        </tbody>
-    </table>
-    {{ paginate({
+
+                            {% if user.hasPpcsRole %}
```

Comment:
Would benefit from an auto format I think.

Reply:
I think we should probably be honouring the Delius restrictions for PPCS users rather than overriding them.  We believe that currently PPCS can see everything, but we should expect/allow that to be specified in Delius.

Reply:
Done!